OpenRA.Game/FieldLoader.cs

```diff
@@ -373,6 +373,28 @@ public static object GetValue(string fieldName, Type fieldType, MiniYaml yaml, M
 
 				return InvalidValueAction(value, fieldType, fieldName);
 			}
+			else if (fieldType == typeof(CVec[]))
+			{
+				if (value != null)
+				{
+					var parts = value.Split(',');
+
+					if (parts.Length % 2 != 0)
+						return InvalidValueAction(value, fieldType, fieldName);
+
+					var vecs = new CVec[parts.Length / 2];
+					for (var i = 0; i < vecs.Length; ++i)
```

Comment:
`i++`, please.

> (15:51:25) abcdefg30 [...] just wanted to know if we have a code style rule about that
(15:51:38) pchote nothing written down, but i++ is used more
(15:51:42) pchote so we should use that